### PR TITLE
Updated nginx configuration for custom-404-page documentation

### PR DESCRIPTION
**Pull request by arjun0108**
The "default" nginx configuration file is the file which has almost all of the server related configurations such as which port will the nginx server listens to, etc.

### CODE REVIEW

**Comment by arjun0108** [comment]
@DirtyF :+1: Maybe, that's right. I'm using nginx version 1.10.3 in Ubuntu-16.04.4 LTS. That's why I was getting confused with the configuration files. 
@kacperduras :+1:  I added the exception for some machines.

File: docs/_tutorials/custom-404-page.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ More info on configuring Apache Error Pages can found in [official documentation
 
 The procedure is just as simple as configuring Apache servers, but slightly different.
 
-Add the following to the nginx configuration file, `nginx.conf`, which is usually located inside `/etc/nginx/` or `/etc/nginx/conf/`:
+The nginx configuration file depends on the system in which it is installed. In most systems, it is the `nginx.conf` file, which is usually located inside `/etc/nginx/` or `/etc/nginx/conf/`. However, in other systems, you would have to look for a `default` nginx configuration file, containing server related information, which is usually located inside `/etc/nginx/sites-available/`. Depending upon the system in which nginx is installed, add the following to the nginx configuration file, _i.e._ either to `nginx.conf` file or to `defualt` file:
```
Comment (DirtyF) [inline docs/_tutorials/custom-404-page.md]: /s/defualt/default

Could we give an example for a specific OS (Ubuntu is fine) and point to external documentation for other systems?

Comment (arjun0108) [reply docs/_tutorials/custom-404-page.md]: Added that in my latest commit

**Comment by DirtyF** [comment]
@jekyllbot: merge +docs

**Comment by DirtyF** [comment]
Thanks @arjun0108 👍

**Comment by arjun0108** [comment]
@DirtyF Always a pleasure contributing something great.

**Pull request merged**